Source/OxyPlot.SkiaSharp/OxyPlot.SkiaSharp.csproj

```diff
@@ -4,14 +4,12 @@
     <TargetFrameworks>net462;netstandard1.3;netstandard2.0</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Copyright>Copyright (c) 2020 OxyPlot contributors</Copyright>
+    <TargetFrameworks>net452;netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
Extra line; would be nice to keep 1.3 if possible I can't remember the situation with Skia

Reply:
Same deal as 1.0 so happy to lose it